proto/kaskada/kaskada/v1alpha/pulsar.proto

```diff
@@ -48,5 +47,5 @@ message PulsarSubscription {
   // subscribing to the pulsar broker.
   string subscription_id = 2;
 
-  google.protobuf.Timestamp last_publish_time = 3;
+  int64 last_publish_time = 3;
```

Comment:
To verify -- this is a backwards incompatible change. We should not make this if we have released any code using the previous definition that could potentially try to talk with the updated version.

Reply:
we have not released any code that uses this.  In fact no current code uses this.